lib/resources/r4/related_person_patch.yaml

```diff
@@ -64,3 +64,37 @@ operations:
         <li>Must include a time component with a timezone when specifying a `period`.</li>
         <li>If `line` is provided, only the first four values are used. Additional values will be ignored.</li>
       </ul>
+
+  - name: test-name-id
+    path: /name/{index}/id
+    operation: test
+    type: string
+    url: https://hl7.org/fhir/R4/element-definitions.html#Element.id
```

Comment:
I think patient test operations are linking to a fhir string link... https://hl7.org/fhir/R4/datatypes.html#string
do we want element.id? im not sure